src/pages/pt-br/install/auto.md

```diff
@@ -37,118 +39,43 @@ yarn create astro
-
-:::tip[Previews online]
-Quer conhecer os temas antes de escolher? Visite: [astro.new](https://astro.new/)
-:::
-
-### Instale as dependências (opcional)
-O assistente de instalação irá se oferecer para executar o comando `install` neste momento para você, etapa opcional.
-
-:::caution
-Se você não deseja fazê-lo neste momento, será necessário [instalar as dependências](#2-instale-as-dependências) após o assistente finalizar e antes de iniciar o seu projeto.
-:::
-
-### Instale qualquer Integração Oficial Astro (opcional)
-Você terá nesse momento a opção de adicionar qualquer [framework de UI adicional](/pt-br/core-concepts/framework-components/) (React, Svelte, Vue, Solid, Preact, Lit) e também qualquer Integração Oficial Astro (Tailwind, Partytown, Sitemap) ao executar `astro add --yes`.
-
-Para selecionar a integração Astro que você deseja incluir em seu projeto, use a teclas direcionais (cima e baixo) para navegar e a barra de espaço para modificar a seleção das opções. Você pode selecionar múltiplos itens de uma vez, ou pode continuar sem selecionar nenhuma integração.
-
-Quando estiver satisfeito com a sua seleção, pressione enter para continuar.
-
-:::note
-Estas integrações e qualquer outra [integração da comunidade Astro](https://astro.build/integrations) podem ser adicionadas posteriormente seguindo as instruções em nosso [guia de integrações](/pt-br/guides/integrations-guide/).
-:::
-
-Após selecionar as integrações, você deverá ver uma mensagem no terminal notificando-o das mudanças que a ferramenta `create-astro` irá aplicar no arquivo `astro.config.mjs` do seu projeto:
-
-```bash
-Astro will make the following changes to your config file:
-```
-
-Esse mensagem irá mostrar qual das integrações escolhidas foram corretamente adicionadas à configuração do seu projeto. (Caso contrário, você pode adicioná-las manualmente depois.)
-
-### Inicialize um repositório `.git` (opcional)
-
-Nesta etapa final, você pode escolher inicializar um repositório git em seu novo diretório. Isto é opcional, mas é útil se você deseja usar a ferramenta [Git](https://git-scm.com/) em seu projeto.
-
-### Próximos passos
+O assistente `create-astro` irá te auxiliar em cada etapa da configuração do seu novo projeto Astro. Você pode executá-lo em qualquer lugar na sua máquina, então não há a necessidade de criar um novo diretório vazio para o seu projeto antes de começar. Se você ainda não tiver nenhum diretório vazio para o seu projeto, o assistente irá te ajudar a criar um para você automaticamente.
 
-Quando o assistente de instalação `create-astro` finalizar, você deverá ver recomendações na sua tela que irão ajudar a completar a configuração e iniciar o seu projeto.
+Se tudo der certo, você verá a mensagem "Ready for liftoff!" ("Pronto para decolagem", em português) seguida de "Next steps" ("Próximos passos", em português) recomendados. `cd` no diretório do seu novo projeto para começar a utilizar Astro.
```

Comment:
I would change the last sentence to make it more clear
```suggestion
Se tudo der certo, você verá a mensagem "Ready for liftoff!" ("Pronto para decolagem", em português) seguida de "Next steps" ("Próximos passos", em português) recomendados. Navegue no diretório do seu novo projeto com o comando `cd` para começar a utilizar Astro.
```

Reply:
I agree, thanks!